scripts/test.js

```diff
@@ -48,6 +58,19 @@ function test(rebuild, browsersArg) {
         fs.copyFileSync(
             path.join(COMPONENTS_FOLDER, component, 'umd', componentPackageName),
             path.join(TESTS_FOLDER, component, componentPackageName));
+
+        if (component === 'form-control') {
+            copyTestDependencies(component, FORM_CONTROL_TEST_DEPENDENCIES);
+        }
+    });
+
+    const formsServer = spawn('node', ['forms-server.js'], { cwd: __dirname });
+    formsServer.stdout.on('data', function (data) {
+        console.log(data.toString());
+    });

```

Comment:
It will be nice if we can avoid always running this server, but unfortunately there's no easy way to spawn it from Karma only if the currently tested component is the form, so let's leave it for now.

Reply:
yes I agree but I don't know how to do it when the form control spec is started so I done it like that ... :/